src/app/app-modules/core/components/app-header/app-header.component.ts

```
@@ -66,14 +67,15 @@ export class AppHeaderComponent implements OnInit, AfterContentChecked {
   
   ngOnInit() {
     this.getUIVersionAndCommitDetails()
      this.license = environment.licenseUrl;
-     const servicePointName: any = localStorage.getItem('tm-servicePointName');
+     const servicePointName: any = this.sessionstorage.getItem('tm-servicePointName');
     this.servicePoint = servicePointName;
-    const tmUserName: any = localStorage.getItem('tm-userName');
+    const tmUserName: any = this.sessionstorage.getItem('tm-userName');
     this.userName = tmUserName;
     this.fetchLanguageSet();
     this.isAuthenticated = sessionStorage.getItem('tm-isAuthenticated') === 'true' ? true : false;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)